misc/plugutils.c

```diff
@@ -1095,6 +1136,8 @@ plug_fork_child (kb_t kb)
       sig_n (SIGTERM, _exit);
       mqtt_reset ();
       kb_lnk_reset (kb);
+      kb_lnk_reset (main);
+      set_main_kb (main);
```

Comment:
why does it need to be stored here again ? wouldn't it be enough just once at the beginning of the scan?

Reply:
that's after the fork; probably just kb_lnk_reset main is sufficient; let me check.

Reply:
I think yes, a link reset should be sufficient.